lib/core/keys.dart

```diff
@@ -130,7 +130,7 @@ String makeKeyPresentable(String key) {
       } else {
         return capitalize(s);
       }
-    }).toList();
+    }).cast<String>().toList();
```

Comment:
Rather than adding a cast, could you instead pass `<String>` as a type argument to `.map` above?

That would avoid some indirection and extra checks in the running app.

Reply:
I had thought that didn't work, but that might have been when I was still being stymied by some unexpected caching semantics for deploying an app over the same version on AppEngine.  Will try again now that I have that straight and use it instead; as you point out, that's a better idea.